superset-frontend/src/components/Loading/index.tsx

```diff
@@ -41,12 +42,16 @@ const LoaderImg = styled.img`
     transform: translate(-50%, -50%);
   }
 `;
-export default function Loading({ position = 'floating' }: Props) {
+export default function Loading({
+  position = 'floating',
+  image = '/static/assets/images/loading.gif',
+}: Props) {
   return (
     <LoaderImg
       className={`loading ${position}`}
       alt="Loading..."
-      src="/static/assets/images/loading.gif"
+      src={image}
+      data-test-id="loading"
```

Comment:
perhaps instead of setting a test id here, we should add the appropriate aria labels, both for a11y and so that RTL can see them.

I think `aria-busy=true` and `aria-label=loading` would be pretty standard, and then you should be able to select it with RTL either using the new aria labels, or with `getByAltText`

Reply:
We did a review here and they told me that.
I did a little research here and it seems to me that aria-busy should be used in the element that will have its content "changed" (in this case it would be the tag that includes this component and not the Loading component).

Same examples:
https://accessibilityresources.org/aria-busy
<img width="764" alt="Captura de Tela 2021-02-17 às 3 15 59 PM" src="https://user-images.githubusercontent.com/1014611/108248807-14baf680-7133-11eb-98c1-03f1bfdbff21.png">

http://sap.github.io/techne/loading-spinner.html#usage-with-other-elements
<img width="958" alt="Captura de Tela 2021-02-17 às 3 16 45 PM" src="https://user-images.githubusercontent.com/1014611/108248905-2bf9e400-7133-11eb-9405-b82c26934807.png">


It seems to me more appropriate to use just `aria-live="polite"` instead of `aria-busy=true` (on Loading component)
https://bitsofco.de/using-aria-live/

Do you agree with that? I'm quite a noob in this accessibility rules lol

Reply:
As for role, boostrap uses `role="status"`
https://getbootstrap.com/docs/4.4/components/spinners/

I will make these changes in case you agree, it’s already easier to merge